grid/chat/workspaces/profile.supr

```diff
@@ -0,0 +1,170 @@
+"""
+Chat Workspaces
+Retrieve workspaces for the user.
+"""
+name = "chat/workspaces"
+version = "1.0.0"
+
+"""
+GetWorkspaces usecase
+"""
+usecase GetWorkspaces {
+  input {
+    """
+    As Bot
+    Boolean representing whether to authorize as bot
+    """
+    asBot boolean
+  }
+
+  result {
+    """
+    Workspaces
+    List of workspaces. Can be workspaces or guilds, depending on the provider capability.
+    """
+    workspaces! [Workspace]!
+
+    """
+    Rate Limit
+    Rate limit details.
+    """
+    rateLimit RateLimit
+  }
+
+  error Error
+
+  example Successful {
+    input {
+      asBot = true
+    }
+
+    result {
+      workspaces = [
+        {
+          id = '1'
+          createdAt = 1643798532134.069
+          name = 'workspace'
+          membersCount = 100
+        },
+        {
+          id = '2'
+          createdAt = 1643216443234.079
+          name = 'workspace 2'
+          membersCount = 4
+          description = 'Test workspace'
+        }
+      ]
+    }
+  }
+
+  example Failed {
+    input {}
+
+    error {
+      title = 'No workspaces found for current user.'
+    }
+  }
+}
+
+model Workspace {
+  """
+  Id
+  Unique identifier of the workspace
+  """
+  id! string!
+
+  """
+  Created At
+  UNIX ms timestamp (e.g. 1095379198750) when the workspace was created
+  """
+  createdAt! number!
+
+  """
+  Name
+  Name of the retrieved workspace
+  """
+  name string
+
+  """
+  Members Count
+  Number of members of the retrieved workspace
+  """
+  membersCount number
+
+  """
+  Icon
+  Icon of the retrieved workspace
```

Comment:
Is it URL or what?

Reply:
Icon hash, I'll edit the description.

Reply:
What is icon hash and how do I use it? IMO only an image URL would make sense here.

Reply:
Ok, I checked and icon hash can be used to construct URL.